portfolio/src/main/webapp/index.html

```diff
@@ -16,7 +16,8 @@
     <h1>Ishani Santurkar</h1>
     <section id="intro-sec">
       <h2>About Me</h2>
-      <img src="/images/me.jpg" alt="An image of Ishani Santurkar" align=right width=400 height=400/>
+      <img src="/images/me.jpg" alt="An image of Ishani Santurkar" id="imgofme" align=right width=400 height=400/>
```

Comment:
According to https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns, let's keep styling in css files and have the HTML only be for structure purposes. 

Same for the div tag below.

Reply:
I couldn't find a CSS analog for the align attribute that work, so I left that one as is.

Reply:
Would `float` work? If not, feel free to go ahead and merge.